build/import/Packages.targets

```diff
@@ -99,7 +99,7 @@
     <PackageReference Update="Microsoft.VisualStudio.HotReload.Components"                            Version="6.0.0-preview.6.21380.4" />
 
     <!-- 3rd party -->
-    <PackageReference Update="Newtonsoft.Json"                                                        Version="12.0.2"/>
+    <PackageReference Update="Newtonsoft.Json"                                                        Version="13.0.1"/>
```

Comment:
Is this the version we're going to ship in 17.0?

Reply:
By "we" I mean VS.

Reply:
Not necessarily, if CPS and it's dependencies moves, this will move further along too.

Reply:
This was the version it was updated to a few weeks back (in VS): https://dev.azure.com/devdiv/DevDiv/_git/VS/pullrequest/340261